gneiss/util.py

```diff
@@ -0,0 +1,117 @@
+def match(x, y, intersect=False):
+    """ Sorts samples in metadata and contingency table in the same order.
+
+    Parameters
+    ----------
+    x : pd.DataFrame
+        Contingency table where samples correspond to rows and
+        features correspond to columns.
+    y: pd.DataFrame
+        Metadata table where samples correspond to rows and
+        explanatory metadata variables correspond to columns.
+    intersect : bool, optional
+        Specifies if only the intersection of samples in the
+        contingency table and the metadata table will returned.
+
+    Returns
+    -------
+    _x : pd.DataFrame
+        Filtered dataframe
+    _y : pd.DataFrame
+        Filtered dataframe
+    """
+    _x = x.sort_index()
+    _y = y.sort_index()
+    if intersect:
```

Comment:
I didn't even think about that.  We shouldn't allow for duplicate ids.  Good catch!